torch/_inductor/codegen/triton.py

```diff
@@ -1327,19 +1327,8 @@ def final_argreduce(buffer, result_var, value, index):
                 )
                 final_argreduce(self.suffix, result_var, accumulator, accumulator_index)
             else:
-                if reduction_type == "min":
-                    updated = f"triton_helpers.minimum({accumulator}, {value})"
-                elif reduction_type == "max":
-                    updated = f"triton_helpers.maximum({accumulator}, {value})"
-                elif reduction_type == "sum":
-                    updated = f"{accumulator} + {value}"
-                elif reduction_type == "prod":
-                    updated = f"{accumulator} * {value}"
-                elif reduction_type == "xor_sum":
-                    updated = f"{accumulator} ^ {value}"
-                else:
-                    raise NotImplementedError(f"reduction_type {reduction_type}")
-
+                combine_fn = ir.get_reduction_combine_fn(reduction_type, src_dtype)
```

Comment:
`arg{max,min}` do not go through this path, and rather re-implement everything. Any chance we can merge those two paths as well?

Reply:
tbh I think `triton_helpers.minimum_with_index` is a lot more readable than generating the whole combine function.

Reply:
That's right. This was more of a

Reply:
That's right. The only thing is that, for that path, we still have two implementations of the same thing, but I guess that's alright.

Reply:
It'd be good to have a note asking people to keep these two in sync if they modify one or the other. No need to do it in this PR tho.